### PR TITLE
Add monitors

### PR DESCRIPTION
**Pull request by NyakoFox**
Monitors allow you to display any image (as long as the URL *ends* with `png`). They're a replacement of image maps.

### CODE REVIEW

File: src/main/java/gay/nyako/nyakomod/NyakoClientMod.java [hunk 1]
```diff
@@ -43,6 +44,7 @@
 
 public class NyakoClientMod implements ClientModInitializer {
 	public static final EntityModelLayer MODEL_DRAGON_LAYER = new EntityModelLayer(new Identifier("nyakomod", "dragon"), "main");
+	public static final EntityModelLayer MODEL_MONITOR_LAYER = new EntityModelLayer(new Identifier("nyakomod", "monitor"), "main");
```
Comment (Ratismal) [inline src/main/java/gay/nyako/nyakomod/NyakoClientMod.java:47]: i love dragons!

File: src/main/resources/data/nyakomod/recipes/monitor.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "GGG",
+    "RIR",
+    "DDD"
+  ],
```
Comment (Ratismal) [inline src/main/resources/data/nyakomod/recipes/monitor.json:7]: this looks like a fair and balanced crafting recipe!

**Review by Ratismal — APPROVED** [review]
LGTM! (Looks Good To Me)

File: src/main/resources/data/nyakomod/recipes/monitor.json (hunk 2 above)
Comment (NyakoFox) [reply src/main/resources/data/nyakomod/recipes/monitor.json:7]: D

**Pull request merged**